### DISCUSSION TITLE
Add categories  /  "albums"

### DISCUSSION THREAD

**Discussion by lemonGon** [jonaswinkler/paperless-ng#1093] [Feature Requests]
I like how granular the filtering is but I would still find useful having the possibility of creating categories and add set of documents to them.
For example I have been living in many countries and I would love to have a category for each of those. or a series of invoices related to the same category (for example photography).

A kind of category tree which would look like this:

Categories
- See All (default)
- Ireland
- Denmark
- Photography
- another category 1
- another category 2

**Comment by michaelkrieger** [comment]
In ScanSnap Organizer and others, they have referred to these as 'filing cabinets' or 'drawers'.  Frankly it's the one thing I think paperless-ng needs.

I have the same challenge.
- Corporation A
- Corporation B
- Home
I've been prefixing some correspondents and tags, but really the organization structure to business records (payables, vendors, tax stuff) and personal is entirely different. "Payables" "Investments" "Bills" "Utilities" could apply to entirely different entities.  There is no point in having an investment property's utility bill mixed in with your personal one.  There is a big difference between payables of Corporation A and Corporation B.

**Just a major category to assign each correspondent, tag, and document type to and then provide a new Documents entry for each major category.**  Yes, in a way this is a 'folder' which is what paperless was trying to get away from, but there's a difference between having a major top-level category that keeps entire entities separate vs a series of subfolders. You'd consume into the main Documents folder and then be able to assign it and add tags/correspondents related only to that 'cabinet'
